readthedocs/projects/tasks.py

```diff
@@ -828,7 +831,7 @@ def is_type_sphinx(self):
 
 # Web tasks
 @app.task(queue='web')
-def sync_files(project_pk, version_pk, hostname=None, html=False,
+def sync_files(project_pk, version_pk, doctype, hostname=None, html=False,
```

Comment:
I'm passing only the `doctype` here rather than the whole `config` object, we only use the doctype, but if someone thinks it's useful passing the `config` I can change it.